tesseract/tesseract_planning/tesseract_motion_planners/test/serialize_test.cpp

```diff
@@ -45,164 +45,175 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 }
 
 TEST(TesseractMotionPlannersTrajoptSerializeUnit, SerializeTrajoptDefaultPlanToXml)  // NOLINT
 {
   // Write program to file
-  TrajOptDefaultPlanProfile plan_profile = getTrajOptPlanProfile();
-  EXPECT_TRUE(toXMLFile(plan_profile, tesseract_common::getTempPath() + "trajopt_default_plan_example_input.xml"));
+  TrajOptDefaultPlanProfile::Ptr plan_profile = getTrajOptPlanProfile();
+  EXPECT_TRUE(toXMLFile(*plan_profile, tesseract_common::getTempPath() + "trajopt_default_plan_example_input.xml"));
 
   // Import file
-  TrajOptDefaultPlanProfile imported_plan_profile = trajOptPlanFromXMLFile(tesseract_common::getTempPath() + ""
-                                                                                                             "trajopt_"
-                                                                                                             "default_"
-                                                                                                             "plan_"
-                                                                                                             "example_"
-                                                                                                             "input."
-                                                                                                             "xml");
+  TrajOptDefaultPlanProfile::Ptr imported_plan_profile = trajOptPlanFromXMLFile(tesseract_common::getTempPath() + ""
+                                                                                                                  "traj"
+                                                                                                                  "opt_"
+                                                                                                                  "defa"
+                                                                                                                  "ult_"
+                                                                                                                  "plan"
+                                                                                                                  "_"
+                                                                                                                  "exam"
+                                                                                                                  "ple_"
+                                                                                                                  "inpu"
+                                                                                                                  "t."
+                                                                                                                  "xm"
+                                                                                                                  "l");
```

Comment:
What's up with this?

Reply:
No idea. I think clang format may have done this.

Reply:
Clang format is doing this, but it is not as bad as before.